app/assets/javascripts/views/vacation_request_form.js

```diff
@@ -53,6 +53,43 @@ App.Views.VacationRequestForm = Backbone.View.extend({
     this.updateFormState();
   },
 
+  onError: function(model, response, options) {
+    // TODO: show error messages to user
+    // console.log(response.responseJSON.errors.base);
+  },
+
+  onSuccess: function(model, response, options) {
+    // Trigger 'sync' on the collection to initiate it's view,
+    // VacationRequestsList, about changes.
+    this.vacationRequests.fetch();
+    // TODO: add some tests
+    // Clear model to set it as a new one,
+    // and initialize it with form data.
+    // Otherwise, the model is initialized with response data and a next save()
+    // will emit PUT request (update) instead of POST (create)
+    this.clearModel();
+    this.fetchFormData();
+  },
+
+  fetchFormData: function () {
```

Comment:
The validation is triggered by `this.model.save()` method, this is the default behaviour of BB models. And the `App.Models.VacationRequest` implements `validation()` method, so the validation is provided.
There an issue #35 that should be resolved later. And during solution implementation for that issue all the validation related errors will be shown to users.

To solve this problem, the following lines should be added to the code:
```JS
this.listenTo(this.model, 'invalid', this.onInvalid);
```
```JS
onInvalid: function(model, response, options) {
   console.log(model.validationError);
}  
```